okrs_2019.md

```diff
@@ -0,0 +1,209 @@
+- * [ ] Produce engaging audiovisual content
+
+### Metrics
+
+| Metric                                  | Goal | Q4 | Q1 | Q2 | Q3 | Q4 |
+| --------------------------------------- | ---- | -- | -- | -- | -- | -- |
+| # of contributors w/ >10 cred           | 15   | 2  |    |    |    |    |
+| # of unique code committers             | 30   | 3  |    |    |    |    |
+| # of core team members                  | 3    | 1  |    |    |    |    |
+| # of Discord community members          | 500  | 61 |    |    |    |    |
+| # of documentation goals accomplished   | 5    | 0  |    |    |    |    |
+
+At the moment, we aren't including metrics around engagement with the
+explanatory content we produce (tracking the number of new members joining the
+community is a good measurement of what we acutally care about, namely
+engagement with the project). However, if we decide to put more emphasis on
+content production, we might update these metrics.
+
+## SourceCred Adoption
+
+Right now, SourceCred is a prototype in search of some real users. Actual users
+will be a great source of feedback on the quality of our attributions, and also
+will bring more people into our community.
+
+### Initiatives
+
+One initiative we’re quite excited about is a planned integration with
+[OpenCollective]. We intend to embed SourceCred results into every open-source
+collective’s page (e.g. here is the [page for Webpack]). This way, collectives
+can thank their leading contributors as well as their financial sponsors. We
+have enthusiastic buy-in from the OpenCollective team, and think this could be
+the start of a great partnership, in addition to bringing more attention and
+users to SourceCred.
+
+[OpenCollective]: https://opencollective.com/
+[page for Webpack]: https://opencollective.com/webpack#contributors
+
+- [ ] Integrate with OpenCollective ([#935])
+  - [ ] Requires API-ifying SourceCred ([#945], [#907])
+  - [ ] Requires SourceCred 'Project Mode' ([#701])
+
+[#935]: https://github.com/sourcecred/sourcecred/issues/935
+[#907]: https://github.com/sourcecred/sourcecred/issues/907
+[#945]: https://github.com/sourcecred/sourcecred/issues/945
+[#701]: https://github.com/sourcecred/sourcecred/issues/701
+
+
+### Metrics
+
+We aren't sure what the right metrics are to track engagement and usage by
+downstream projects. For now, we'll measure "engagment", which is defined as:
+the project has taken a public action to use SourceCred, e.g. by linking to the
+cred attribution in their README, adding cred configuration files to their repository,
+or integrating a cred bot on GitHub. Actions that can happen without direct engagement
+from the project (e.g. listing their cred on OpenCollective) don't count.
+
+For the purposes of the metrics below, "medium-sized projects" have at least 30 users
+in their cred attribution, and "large-sized projects" have at least 100.
+
+We'd like to see crypto projects start to use SourceCred as a way to transparently
```

Comment:
> We'd like to see crypto projects start to use SourceCred as a way to
> transparently distribute tokens to contributors. For purposes of the
> metric, we're tracking public announcements or active distributions by
> well-known projects (i.e. no unknown ICOs).

I have some concerns about the incentives that this creates. Let me try
to express them clearly and get back to you.

Reply:
Ack.

Reply:
On a perhaps different topic, I don't love this measure of "engagement." I understand the trade-offs at hand (I think), but it creates an incentive for vocal participants over real participation. Both are key (the former for visibility and users) but I think engagement must include the latter, as the sole measure of SourceCred's actual usefulness to projects (vs whether it sounds good to people, or helps them sell).

Potential solutions:
  - (1) A quick callback to SC from downstream projects that helps them automate tracking of SC use: how often they integrate it to a page, how often that page is viewed, etc (super centralized and pretty bad)
 - (2) A more qualitative metric such as a measure of engagement derived from a standard questionnaire SC provides users on a quarterly basis (participation rate there is a good measure onto itself)
- (3) Some tracking of discussions surrounding specific downstream work in the SC forums themselves (hard to measure)
- (4) Simply an acknowledgement that this is a flawed metric and will be upgraded once early projects are onboarded.

I would opt for 2 or 4 to start

Reply:
I don't understand your concerns around engagement, or how it prioritizes vocal users. Here, a project is described as "engaged" if there is externally-visible evidence that the project has made a deliberate decision to use SourceCred. An example of what this might look like is, maybe as we add .credconfig files that allow projects to tune their weights and heuristics, and we count how many projects have merged these configs.

I do like the idea of coming up with better engagement metrics. The quarterly questionnaire is a nice idea. For now I'm inclined to leave this as it is, for me every metric in the doc is implicitly open to being upgraded.

Reply:
Fair enough, and I really like the .credconfig idea, which signals they've actually meaningfully engaged with SC. My issue with externally-visible signs as I initially read them (ie mention of SC in some public page for the project) is that it sets it a shallow bar as to what we consider an engaged user (in a way that the .credconfig file does less so). In time the hope is that using SC becomes a best practice in OSS development, and simply saying your project uses SC isn't good enough, but we're on the same page!

Henri